ConfigSpace/hyperparameters/normal_float.pyx

```diff
@@ -207,7 +207,9 @@ cdef class NormalFloatHyperparameter(FloatHyperparameter):
                                            q=q_int, log=self.log)
 
     def is_legal(self, value: Union[float]) -> bool:
-        return isinstance(value, float) or isinstance(value, int)
+        return (isinstance(value, float) or isinstance(value, int))  and \
```

Comment:
Can we make this `isinstance(value, (float, int, np.number))`?

Reply:
Done.